[2024-02-12T23:00:42.751+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.load_data_into_the_table scheduled__2024-02-12T22:30:00+00:00 [queued]>
[2024-02-12T23:00:42.767+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.load_data_into_the_table scheduled__2024-02-12T22:30:00+00:00 [queued]>
[2024-02-12T23:00:42.768+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-12T23:00:42.787+0000] {taskinstance.py:2191} INFO - Executing <Task(SnowflakeOperator): load_data_into_the_table> on 2024-02-12 22:30:00+00:00
[2024-02-12T23:00:42.796+0000] {standard_task_runner.py:60} INFO - Started process 349 to run task
[2024-02-12T23:00:42.801+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'load_data_into_the_table', 'scheduled__2024-02-12T22:30:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmppfq7jeh6']
[2024-02-12T23:00:42.803+0000] {standard_task_runner.py:88} INFO - Job 148: Subtask load_data_into_the_table
[2024-02-12T23:00:42.896+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.load_data_into_the_table scheduled__2024-02-12T22:30:00+00:00 [running]> on host 3bd3b2605e24
[2024-02-12T23:00:43.016+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='load_data_into_the_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T22:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T22:30:00+00:00'
[2024-02-12T23:00:43.018+0000] {sql.py:276} INFO - Executing: 
                    COPY INTO Bangkok from @s3_stage file_format=csv_format;

            
[2024-02-12T23:00:43.035+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-12T23:00:43.049+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-12T23:00:43.050+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.8.18, Platform: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.34
[2024-02-12T23:00:43.052+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-02-12T23:00:43.510+0000] {cursor.py:1028} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-02-12T23:00:43.617+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T23:00:43.619+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T23:00:43.620+0000] {sql.py:450} INFO - Running statement: COPY INTO Bangkok from @s3_stage file_format=csv_format;, parameters: None
[2024-02-12T23:00:43.621+0000] {cursor.py:1028} INFO - query: [COPY INTO Bangkok from @s3_stage file_format=csv_format;]
[2024-02-12T23:00:44.560+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T23:00:44.561+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T23:00:44.562+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-12T23:00:44.563+0000] {snowflake.py:400} INFO - Rows affected: 1
[2024-02-12T23:00:44.564+0000] {snowflake.py:401} INFO - Snowflake query id: 01b24f84-3200-fe68-0007-a44e00010796
[2024-02-12T23:00:44.565+0000] {cursor.py:1028} INFO - query: [COMMIT]
[2024-02-12T23:00:44.804+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T23:00:44.805+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T23:00:44.806+0000] {connection.py:718} INFO - closed
[2024-02-12T23:00:44.876+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2024-02-12T23:00:44.972+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=load_data_into_the_table, execution_date=20240212T223000, start_date=20240212T230042, end_date=20240212T230044
[2024-02-12T23:00:45.025+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T23:00:45.056+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
